### PR TITLE
Jordan/1105 steak and atoms

### LINKED ISSUE #1105: "You do not have any ATOMs to delegate." confusing while using STEAK
UI Version: 0.9.3

Description:

I've been staking with STEAK but just got the error message, "You do not have any ATOMs to delegate."  The terminology is confusing.

### PR DESCRIPTION
**Pull request by jbibla**
Closes #1105 

Description:

- attempted to replace all uses of Steak or Atom with a config variable
- minor style updates as i noticed them while hunting for steak and atom
- changed the casing (ATOMs vs Atoms, STEAK vs Steak)

❤️ Thank you!

### CODE REVIEW

File: test/unit/specs/components/staking/__snapshots__/PageBond.spec.js.snap [hunk 1]
```diff
@@ -176,7 +176,7 @@ exports[`PageBond disables bonding if not connected 1`] = `
                         class="bond-value__input tm-field"
                         disabled="disabled"
                         id="new-unbonded-atoms"
-                        placeholder="Steaks"
+                        placeholder="0"
```
Comment (okwme) [inline test/unit/specs/components/staking/__snapshots__/PageBond.spec.js.snap]: placeholder="0" ?! /1 ? 1?!?!?!

File: test/unit/specs/components/staking/__snapshots__/PageBond.spec.js.snap [hunk 2]
```diff
@@ -662,7 +662,7 @@ exports[`PageBond has the expected html structure 1`] = `
...
```
Comment (okwme) [inline test/unit/specs/components/staking/__snapshots__/PageBond.spec.js.snap]: ?!?!?!

File: test/unit/specs/components/staking/__snapshots__/PageBond.spec.js.snap [hunk 3]
```diff
@@ -1167,7 +1167,7 @@ exports[`PageBond shows a message if there are revoked candidates 1`] = `
...
```
Comment (okwme) [inline test/unit/specs/components/staking/__snapshots__/PageBond.spec.js.snap]: !!!!!

File: test/unit/specs/components/staking/__snapshots__/PageBond.spec.js.snap (hunk 1 above)
Comment (jbibla) [reply test/unit/specs/components/staking/__snapshots__/PageBond.spec.js.snap]: i think having `Steaks` or `Atoms` would be misleading the user to think they should enter a denomination when really the input is expecting a number.

File: app/src/renderer/components/staking/PageStaking.vue [hunk 4]
```diff
@@ -109,39 +116,40 @@ export default {
...
         },
         {
-          title: "Total Votes",
+          title: `Total ${this.bondingDenom}`,
           value: "voting_power",
-          tooltip: `The delegate stakes this many ${
+          tooltip: `Total number of ${
             this.bondingDenom
-          }s on the network.`,
+          } the validator has on The Cosmos Hub`,
```
Comment (fedekunze) [inline app/src/renderer/components/staking/PageStaking.vue]: Note: will eventually have to change "The Cosmos Hub" to a variable with the name of the zone

**Pull request merged**